File: Ashokai.novaextension/_resources/Jokes/joke.php
<? class Joke {
  private $question;
  private $answer;

  public function __construct($question, $answer) {
    $this->question = $question;
    $this->answer = $answer;
  }

  public function tellJoke()
    echo $this->question . PHP_EOL;
    sleep(2);
    echo $this->answer . PHP_EOL;
  }
}

$joke = new Joke(
  "Why don't skeletons fight each other?",
  "They don't have the guts!"
);

$joke->tellJoke(); ?>

File: Ashokai.novaextension/_resources/Jokes/joke.py
import time

class Joke:
    def __init__(self, question, answer):
        self.question = question
        self.answer = answer

    def tell_joke(self):
        print(self.question)
        time.sleep(2)
        print(self.answer)

# Create an instance of the Joke class
joke = Joke(
    "Why don't cats play poker in the wild?",
    "Too many cheetahs!"
)

# Call the tell_joke method to share the joke
joke.tell_joke()
